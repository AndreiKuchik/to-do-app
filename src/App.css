.App {
  text-align: center;
}
.App-logo {
    margin: 0 0 1rem 0;
    max-width: 95%;
    border-radius: 12px;
    text-align: center;
}
