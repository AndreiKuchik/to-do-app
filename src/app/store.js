import { configureStore } from '@reduxjs/toolkit'

import toDoListReducer from '../features/toDoListSlice'

export default configureStore({
  reducer: {
    toDoList: toDoListReducer
  },
})
