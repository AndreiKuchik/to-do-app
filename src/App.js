import './App.css';
import React from 'react';
import AddPostForm from './features/ToDoForm/AddToDoForm';
import Header from './components/Header';
import ToDos from './features/ToDos/ToDos';

function App() {
  return (
    <div className='App'>
      <Header/>
      <AddPostForm/>
      <ToDos/>
    </div>
  );
}

export default App;
