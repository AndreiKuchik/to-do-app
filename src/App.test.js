import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import App from './App';
import store from './core/store';

describe('Jest Snapshot testing suite', () => {
  it('Matches DOM Snapshot', () => {
    const domTree = renderer
      .create(
        <Provider store={store}>
          <App />
        </Provider>,
      )
      .toJSON();
    expect(domTree).toMatchSnapshot();
  });
});
