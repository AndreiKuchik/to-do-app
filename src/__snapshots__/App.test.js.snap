// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jest Snapshot testing suite Matches DOM Snapshot 1`] = `
<div
  className="App"
>
  <header
    className="todo-image"
  >
    <img
      alt="Logo"
      className="App-logo"
      src="logo.png"
    />
  </header>
  <div
    className="card toDoForm"
  >
    <form>
      <div
        className="newToDoControls"
      >
        <div
          className="newToDoControl"
        >
          <input
            className="todo-input"
            onChange={[Function]}
            placeholder="Please type your ToDO"
            type="text"
            value=""
          />
        </div>
        <div
          className="newToDoActions"
        >
          <button
            className="submit-todo-input"
            disabled={true}
            onClick={[Function]}
            type="button"
          >
            Save ToDo
          </button>
        </div>
      </div>
    </form>
  </div>
  <div
    className="card toDos"
  />
</div>
`;
